core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -1663,7 +1663,7 @@ public String getDisplayName() {
         return Messages.Hudson_DisplayName();
     }
 
-    public synchronized List<JDK> getJDKs() {
+    public List<JDK> getJDKs() {
         if(jdks==null)
```

Comment:
So I believe the only way this could be null is if a configuration was saved without the <jdks> entry and then reloaded.
I don't see how that could happen - but there may be some old version of Jenkins that could have done it.

Would probably make sense to modify a `readResolve()` to set it to an empty list and remove this check and the theoretical possibility?